### DISCUSSION TITLE
How can I run the Merlin examples on CPU?

### DISCUSSION THREAD

**Discussion by michalgenus** [NVIDIA-Merlin/Transformers4Rec#772] [Q&A]
Hi, I couldn't find a documentation on how to run the examples here on CPU instead of GPU.

I'm running the examples through Jupyter environment on Nvidia's docker (nvcr.io/nvidia/merlin/merlin-pytorch:23.12).
Specifically, this example: examples/end-to-end-session-based.

The functions are configured to run through GPU. 
For example, this function: save_time_based_splits, from transformers4rec.utils.data_utils.
Where can I find its documentation, and does it have a parameter that enables it to run on CPU?

Thanks

**Comment by rnyak** [comment]
did you try setting `cpu=True` [here](https://github.com/NVIDIA-Merlin/Transformers4Rec/blob/d0cce61b988a1e923545f94ac22499fb57928d18/transformers4rec/utils/data_utils.py#L177)
